package.json

```diff
@@ -119,6 +119,10 @@
       "python": {
         "distName": "cdktf-cdktf-provider-null",
         "module": "cdktf_cdktf_provider_null"
+      },
+      "dotnet": {
+        "namespace": "HashiCorp.Cdktf.ProviderNull",
+        "packageId": "HashiCorp.Cdktf.ProviderNull"
```

Comment:
Does this naming make sense to you @jsteinich?

Reply:
I would probably go with `HashiCorp.Cdktf.NullProvider` or `HashiCorp.Cdktf.Providers.Null` so it reads better.